js/w3c/headers.js

```diff
@@ -535,6 +535,12 @@ define(
                     msg.pub("error", "ReSpec does not support automated SotD generation for TAG findings, " +
                                      "please specify one using a <code><section></code> element with ID=sotd.");
                 }
+                // experimental style info
+                if (conf.useExperimentalStyles) {
+                    $("html>head").prepend($('<meta name="viewport" content="width=device-width, initial-scale=1, width = content-width, initial-scale = 1.0, shrink-to-fit=no">'));
```

Comment:
I can't remember exactly, but adding this after the fact might not actually work on mobile. 
Also, should probably check if: 
```
var hasViewportMeta = doc.head.querySelector("meta[name=viewport]") ? true : false;
if (conf.useExperimentalStyles && !hasViewportMeta) { ...}
```
Before adding another on - as having two might result in strangeness.